src/OSLikeStuff/task_scheduler.cpp

```diff
@@ -39,6 +43,13 @@ struct StatBlock {
 #endif
 		average = (average + v) / 2;
 	}
+	void reset() {
+#if SCHEDULER_DETAILED_STATS
+		min = std::numeric_limits<double>::infinity();
+		max = -std::numeric_limits<double>::infinity();
```

Comment:
why is min positive and max negative?

Reply:
They're initial values - for min the algorithm is going to always make this value smaller so it needs to start by overestimating